code/modules/clothing/head/hardhat.dm

```diff
@@ -33,5 +31,6 @@
 /obj/item/clothing/head/hardhat/red/atmos
 	name = "atmospheric firefighter helmet"
 	desc = "An atmospheric firefighter's helmet, able to keep the user protected from heat and fire."
+	max_heat_protection_temperature = FIRE_HELMET_MAX_HEAT_PROTECTION_TEMPERATURE + 15000
```

Comment:
This kind of use of defines kinda overrules their purpose. Consider creating more macros and redefining the max?

Reply:
Why? Fire suit is a base case and we got 2-3 exceptions.

Reply:
~~The point of defines is defeated if they're generally turned into a magic number generator.~~